picard/script.py

```diff
@@ -846,6 +846,17 @@ def func_ne_any(parser, x, *args):
     """
     return func_not(parser, func_eq_all(parser, x, *args))
 
+def func_title(parser, text):
+    """ Title-case a text """
```

Comment:
Please make a more complete description of the function, including an example.

Also add a test for it in https://github.com/metabrainz/picard/blob/master/test/test_script.py